ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -230,6 +230,9 @@ echo -e "\nPreparing files for next steps in SHARED_DIR..."
 cp "${INSTALL_DIR}/auth/kubeconfig" "${SHARED_DIR}/"
 cp "${INSTALL_DIR}/auth/kubeadmin-password" "${SHARED_DIR}/"
 
+# Copy coreos stream file so the observer pod can check if the correct live image was booted
+oinst coreos print-stream-json > "${SHARED_DIR}/coreos-stream.json"
```

Comment:
can you add these two lines in the other install scripts (bm upi and bm ipi)? no tests needed for now, but having them is enough.